hobbyloop-api/src/main/java/hobbyloop/backend/api/infra/global/oauth2/handler/OAuth2AuthenticationSuccessHandler.java

```diff
@@ -21,7 +20,7 @@
 @Component
 @RequiredArgsConstructor
 @Transactional
```

Comment:
```suggestion
@Transactional(readOnly = true) 
```
로 하고 밑에서 필요할 때 `@Transactional` 붙이는 방향으로 기존 방법 통일성 가져가죠